Makefile

```diff
@@ -23,7 +23,7 @@ PLUGIN_BINARY := ./bin/firehose.so
 build: $(PLUGIN_BINARY)
 
 $(PLUGIN_BINARY): $(SOURCES)
-	PATH=${PATH} golint ./plugins ./firehose
+	PATH=${PATH} ./bin/golangci-lint run -c golint.yml
```

Comment:
My question/concern is- can you run and install this locally? The make targets are all meant to be runnable on a dev machine. The executable path makes me suspect it only works in GitHub actions..